unit-tests/func/mur/l500/test-require-vga-streaming.cpp

```diff
@@ -0,0 +1,48 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#test:tag max-usable-range
+//#test:device L500*
+
+#include <thread>
+#include <chrono>
+#include "../../func-common.h"
+
+// Test group description:
+//       * This tests group verifies depth sensor max usable range option restrictions,
+//         according to [RS-8358]
+//
+// Current test description:
+//       * Enabling Max Usable Range option while streaming require Sensor Mode = 'VGA' 
+//         If not an exception should be thrown
+//       
+// Note: * L515 specific test
+//       * Test will pass if no L515 device is connected
+//       * Test will be performed on the first L515 device detected.
+
+TEST_CASE( "Max Usable Range option require VGA sensor mode while streaming", "[max-usable-range]" )
+{
+    auto depth_sensor
+        = find_first_supported_depth_sensor_or_exit( "L515", RS2_OPTION_ENABLE_MAX_USABLE_RANGE );
+    auto mur = depth_sensor.as< rs2::max_usable_range_sensor >();
+    REQUIRE(mur);
+    mur.set_option( RS2_OPTION_VISUAL_PRESET, RS2_L500_VISUAL_PRESET_MAX_RANGE );
+    auto profiles = mur.get_stream_profiles();
+    // Find a profile that sensor mode != VGA
+    auto non_vga_depth_profile
```

Comment:
You can just ask for a specific resolution, no?

Reply:
But OK otherwise

Reply:
Yes I can, but since I don't care what it will be, I want the first one that apply my request.